CONTRIBUTING.md

```diff
@@ -15,7 +15,7 @@ Jekyll (a static site generator) using scripts from `./cmd` and formatting from
 
 All changes to the repository should be made via PR
 ([OSPS-AC-03](https://baseline.openssf.org/#osps-ac-03)).  In addition to a clear
-title and descriptive commit message, PRs MUST meet the following criterion:
+title and descriptive commit message, PRs MUST meet the following control:
```

Comment:
Neither the old nor the new is correct here. 😄 This should be "criteria".

Reply:
```suggestion
title and descriptive commit message, PRs MUST meet the following criteria:
```